python/semantic_kernel/utils/tracing/const.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft. All rights reserved.
+#
+# Constants for tracing activities with semantic conventions.
+
+# Activity tags
+SYSTEM = "gen_ai.system"
+OPERATION = "gen_ai.operation.name"
+MODEL = "gen_ai.request.model"
+MAX_TOKEN = "gen_ai.request.max_tokens"  # nosec
```

Comment:
Just curious: what is the error message that we are ignoring?

Reply:
The tool thought this was a hard-coded security token